src/lkl/posix-host.c

```diff
@@ -612,7 +610,7 @@ struct lkl_host_operations sgxlkl_host_ops = {
     .tls_free = tls_free,
     .tls_set = tls_set,
     .tls_get = tls_get,
-    .time = time_ns,
+    .time = enclave_nanos,
```

Comment:
The system `time` function should produces the seconds elapsed since epoch. With the current implementation, the `time` function returns the seconds since we launch sgx-lkl.

Reply:
What current implementation?

Reply:
With these changes, time(NULL) returns seconds since epoch, not seconds since the launch of sgx-lkl.

Reply:
This does not matter.  The time function is a time source provided to Linux's hrtimer infrastructure.  It starts at an arbitrary point and increments monotonically, modelling a hardware time source.  The kernel records the initial value as the offset for uptime and a different (mutable) offset to use for the wall-clock time.  The fact that the old time source started at an arbitrary point in the past and this one starts at a different arbitrary point in the past does not matter to the consumers.